apps/web/src/components/SegmentDisplay.module.css

```
@@ -0,0 +1,55 @@
+}
+
+.digit {
+  display: block;
+  height: 1em;
+  width: auto;
+  aspect-ratio: 24 / 44;
+}
+
+/* Ghost segment — the faint "8" always visible behind every digit. */
+.seg {
+  fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)